src/composables/aeSdk.ts

```diff
@@ -204,6 +204,22 @@ export function useAeSdk() {
     );
   }
 
+  async function resetConnectedDapps() {
```

Comment:
This code was copied 1:1 from the `wallet.js` file without changes. But I have a feeling that something is wrong here. The `_clients` property is typed on the SDK side as `Map<string, RpcClientsInfo>`. So the `aepp` is the `RpcClientsInfo` and this interface has no `info` property.

@davidyuk @CedrikNikita

Reply:
`info` was removed in 12.0.0 because it is not needed on sdk side

> [RpcClient: doesn't contain aepp info anymore](https://github.com/aeternity/aepp-sdk-js/tree/7126ad02166747a75c8462b13386774cfc003124/docs/guides/migration/12.0.0.md#rpcclient-doesnt-contain-aepp-info-anymore)
Get aepp info in onConnection callback, and store somehow to use later.

https://github.com/aeternity/aepp-sdk-js/commit/010ebbb963ac8058ed74bff1fd01ad2e243eb3ab

Soooo, this code probably is not working